packages/teams-js/src/public/liveShareHost.ts

```diff
@@ -4,149 +4,184 @@ import { FrameContexts } from './constants';
 import { runtime } from './runtime';
 
 /**
- * @hidden
- * Allowed roles during a meeting.
+ * APIs involving Live Share, a framework for building real-time collaborative apps.
+ * For more information, visit https://aka.ms/teamsliveshare
  *
- * @beta
+ * @see LiveShareHost
  */
-export enum UserMeetingRole {
-  guest = 'Guest',
-  attendee = 'Attendee',
-  presenter = 'Presenter',
-  organizer = 'Organizer',
-}
-
-/**
- * @hidden
- * State of the current Live Share sessions backing fluid container.
- *
- * @beta
- */
-export enum ContainerState {
+export namespace liveShare {
   /**
-   * The call to `LiveShareHost.setContainerId()` successfully created the container mapping
-   * for the current Live Share session.
+   * @hidden
+   * The meeting roles of a user.
+   * Used in Live Share for its role verification feature.
```

Comment:
Is there documentation for the role verification feature? If so, it would be useful to link that here.

Reply:
Sure, @huntj88 please add the following link to the doc: https://learn.microsoft.com/microsoftteams/platform/apps-in-teams-meetings/teams-live-share-capabilities?tabs=javascript#role-verification-for-live-data-structures